reporting/transformer_example/src/transformer_example.py

```diff
@@ -50,82 +39,82 @@ class ElasticsearchRecord(object):
     doc = attrib()
 
 
-def _extract_hybrid_record(raw_record):
-    return HybridRecord(**raw_record)
+# Moving records through the pipeline ------------------------------------------
+def hybrid_record_to_data_dict(s3, hybrid_record):
+    """
+    takes an VHS/S3 location and returns its data as a record in ES format
 
+    Parameters
+    ----------
+    s3 : boto3.client
+    hybrid_record : HybridRecord
 
-def _get_record_from_s3(s3, object_location):
-    obj = s3.get_object(Bucket=object_location.namespace, Key=object_location.key)
-    return obj["Body"].read().decode("utf-8")
-
-
-def _build_es_record(s3, hybrid_record):
-    s3_record = _get_record_from_s3(s3, hybrid_record.location)
-    return ElasticsearchRecord(id=hybrid_record.id, doc=s3_record)
+    Returns
+    -------
+    data_dict : dict
+        object data in malleable dict format
+    """
+    object_location = hybrid_record.location
+    s3_object = s3.get_object(Bucket=object_location.namespace, Key=object_location.key)
+    record = s3_object["Body"].read().decode("utf-8")
+    record_data = json.loads(record)["data"]
+    return record_data
```

Comment:
Suggestion: don’t combine these two functions into one. Previously you had a function that could read from S3, another that unpacked that as an ElasticsearchRecord. Keeping them separate makes it easier to reuse parts of them later.

Reply:
Also though possibly @alexwlchan would object, having separate functions with descriptive names goes some way to having the code be self documenting.